pkg/ccl/crosscluster/logical/dead_letter_queue_test.go

```diff
@@ -212,6 +228,27 @@ func TestDLQClient(t *testing.T) {
 		fmt.Sprintf(`SELECT schema, name, values FROM [SHOW ENUMS FROM %s.%s]`, defaultDbName, dlqSchemaName), enumRow)
 	sqlDB.CheckQueryResults(t,
 		fmt.Sprintf(`SELECT schema, name, values FROM [SHOW ENUMS FROM %s.%s]`, dbAName, dlqSchemaName), enumRow)
+}
+
+func TestDLQLogging(t *testing.T) {
```

Comment:
What's the distinction between the `TestDLQLogging` and `TestLoggingDLQClient`? I _think_ you meant to delete the latter?

Reply:
The names are pretty confusing, `TestLoggingDLQClient` is a test for `loggingDeadLetterQueueClient`, maybe it's better to rename it to `noopDeadLetterQueueClient` if we still want to keep it, otherwise we can just remove it since it's not being used anywhere? 
`TestDLQCreation` tests `deadLetterQueueClient.Create()` and `TestDLQLogging ` tests `deadLetterQueueClient.Log()`